### DISCUSSION TITLE
Where is the function('collate_line()' in 'extract_text()') in pdfplumber==0.8.0

### DISCUSSION THREAD

**Discussion by Godlikemandyy** [jsvine/pdfplumber#857] [Q&A]
I can't find the collate_line() function in the extract_text() function in version 0.8.0. Where can I find it? Has the structure of the extract_text() function changed in the new version?

**Comment by jsvine** [comment]
`collate_line` is available via `from pdfpumbler.utils import collate_line`; you can also find the code itself in `pdfplumber/utils/text.py`.
